java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -6941,6 +6941,97 @@ public void xclaim_return_failures(BaseClient client) {
     //         assertInstanceOf(RequestException.class, executionException.getCause());
     //     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void xautoclaim(BaseClient client) {
+        String minVersion = "6.2.0";
+        assumeTrue(
+                REDIS_VERSION.isGreaterThanOrEqualTo(minVersion),
+                "This feature added in redis " + minVersion);
+        boolean isVersion7OrAbove = REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0");
+        String key = UUID.randomUUID().toString();
+        String groupName = UUID.randomUUID().toString();
+        String zeroStreamId = "0-0";
+        String consumer = UUID.randomUUID().toString();
+
+        // Add 4 stream entries for consumer
+        String streamid_0 = client.xadd(key, Map.of("f1", "v1" /*, "f2", "v2"*/)).get();
```

Comment:
Please use LinkedHashMap instead of Map.of() throughout so that there's consistent ordering during xadd.

Reply:
... only if has effect on the command being tested

Reply:
will ignore this one for now

Reply:
This is fine to skip because there's only one entry but we should really just use LinkedHashMap by default to avoid test debugging.